connect/src/com/telenor/connect/ConnectSdk.java

```diff
@@ -106,7 +105,7 @@ public static synchronized void authenticate(
             final Map<String, String> parameters,
             final int requestCode) {
         Validator.sdkInitialized();
-        sdkProfile.onStartAuthorization(parameters, new SdkProfile.OnStartAuthorizationCallback() {
+        sdkProfile.onStartAuthorization(parameters, new SdkProfile.OnStartAuthenticationCallback() {
```

Comment:
It's right that openid connect is also authentication, but since the flow is called "authorization code flow" I think we should just stick with authorization.